__tests__/app.test.js

```diff
@@ -171,6 +172,70 @@ describe("app GET requests", () => {
         })
     })
 
+    describe("GET /api/articles/:article_id/comments", () => {
+    // Functionality tests
+
+        //test variable for parametric endpoint
+        const article_id = 1;
```

Comment:
You should declare variables locally in each test, try to make each test completely isolated from others